lib/omnibus-ctl.rb

```diff
@@ -322,7 +322,7 @@ def scary_cleanse_warning(*args)
           sleep 60
         rescue Interrupt
           log ""
-          exit 0
+          Kernel.exit 1
```

Comment:
THese seemed a safe exit code change to make, because this is handling for when the operator explicitly does not pass the `--yes` argument that will bypass the wait period

Interrupted CLIs typically exit non-zero.